config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root to: 'home#show'
   resources :users, only: [:new, :create, :show, :update]
+  post :sms, to: 'sms/users#update'
```

Comment:
i like that! that sounds good to me. i agree it's a good idea to keep it separate from the api 👍